cmd/cloud-run/rotate-service-account/rotateserviceaccount.go

```diff
@@ -58,17 +58,20 @@ func main() {
 
 	projectID, err = metadata.ProjectIDWithContext(ctx)
 	if err != nil {
-		mainLogger.LogCritical("failed to retrieve GCP Project ID, error: " + err.Error())
+		const errMsg = "failed to retrieve GCP Project ID, error: %s"
+		mainLogger.LogCritical(errMsg, err.Error())
```

Comment:
Can we use such syntax? Defining a constant, for each format string going to logs looks very odd. We are not going to reuse this constant in other places and create it with different values in multiple scopes.
```suggestion
		mainLogger.LogCritical("failed to retrieve GCP Project ID, error: %s", err.Error())
```

Reply:
I agree, I will prepare separate PR with fix of this.

Reply:
I think the linter will say that the string is not constant, that's why this change was introduced.

Reply:
I will check that

Reply:
I believe the linter was complaining because we were first calling fmt.Sprintf which returns string. That string then was passed as only argument to mainLogger.LogCritical which in fact uses fmt.Sprintf. The linter expect us to provide format and variables to fmt.Sprintf.